tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun_test.go

```diff
@@ -292,6 +292,10 @@ var aPipelineLoop = &pipelineloopv1alpha1.PipelineLoop{
 		PipelineRef:           &v1beta1.PipelineRef{Name: "a-pipeline"},
 		IterateParam:          "current-item",
 		IterateParamSeparator: "separator",
+		ServiceAccountName:    "default",
+		PodTemplate: &v1beta1.PodTemplate{
+			HostNetwork: false,
```

Comment:
Thanks @ScrapCodes, do we have a test for podTemplate to check is the new pipelineloop object is generated as expected? Also, I saw you added `TaskRunSpecs` as well, can you add a small unit test to make sure it can pass the new `TaskRunSpecs` as expected? Thanks.

Reply:
Thank you @Tomcli, sure!